ibm_cloud_sdk_core/base_service.py

```diff
@@ -373,14 +372,13 @@ def prepare_request(self,
         params = cleanup_values(params)
         request['params'] = params
 
-        if sys.version_info >= (3, 0) and isinstance(data, str):
```

Comment:
We are not supporting Python 2 as far as I know, so I removed the version check.